<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tiffany</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="centralWidgetLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="Bookmarks" name="bookmarks">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="Viewer" name="viewer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="scrollAreaLayout"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAct"/>
    <addaction name="insertAct"/>
    <addaction name="replaceAct"/>
    <addaction name="separator"/>
    <addaction name="saveFilesAct"/>
    <addaction name="saveToAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="selectAllAct"/>
    <addaction name="selectOddAct"/>
    <addaction name="selectEvenAct"/>
    <addaction name="separator"/>
    <addaction name="deleteAct"/>
    <addaction name="blankAct"/>
    <addaction name="fontAct"/>
    <addaction name="separator"/>
    <addaction name="rotateCWAct"/>
    <addaction name="rotateCCWAct"/>
    <addaction name="rotate180Act"/>
    <addaction name="mirrorHorizAct"/>
    <addaction name="mirrorVertAct"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="fitToWindowAct"/>
    <addaction name="fillWindowAct"/>
    <addaction name="fitWidthAct"/>
    <addaction name="fitHeightAct"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="pointerAct"/>
    <addaction name="dropperAct"/>
    <addaction name="floodAct"/>
    <addaction name="pencilAct"/>
    <addaction name="eraserAct"/>
    <addaction name="deskewAct"/>
    <addaction name="despeckleAct"/>
    <addaction name="devoidAct"/>
   </widget>
   <widget class="QMenu" name="menu_Stroke">
    <property name="title">
     <string>&amp;Stroke</string>
    </property>
    <addaction name="pix1Act"/>
    <addaction name="pix4Act"/>
    <addaction name="pix8Act"/>
    <addaction name="pix12Act"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAct"/>
    <addaction name="aboutQtAct"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Stroke"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/file_open.svg</normaloff>:/images/assets/file_open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="insertAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/file_add.svg</normaloff>:/images/assets/file_add.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert</string>
   </property>
  </action>
  <action name="replaceAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/file_replace.svg</normaloff>:/images/assets/file_replace.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Replace</string>
   </property>
  </action>
  <action name="saveFilesAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/file_save.svg</normaloff>:/images/assets/file_save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save changed files</string>
   </property>
  </action>
  <action name="saveToAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/file_saveas.svg</normaloff>:/images/assets/file_saveas.svg</iconset>
   </property>
   <property name="text">
    <string>Save &amp;To</string>
   </property>
   <property name="toolTip">
    <string>Save changed files to directory</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="selectAllAct">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
  </action>
  <action name="selectOddAct">
   <property name="text">
    <string>Select &amp;Odd</string>
   </property>
  </action>
  <action name="selectEvenAct">
   <property name="text">
    <string>Select &amp;Even</string>
   </property>
  </action>
  <action name="deleteAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/delete.svg</normaloff>:/images/assets/delete.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="rotateCWAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/rotate_CW.svg</normaloff>:/images/assets/rotate_CW.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Rotate CW</string>
   </property>
  </action>
  <action name="rotateCCWAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/rotate_CCW.svg</normaloff>:/images/assets/rotate_CCW.svg</iconset>
   </property>
   <property name="text">
    <string>Rotate CCW</string>
   </property>
   <property name="toolTip">
    <string>Rotate CCW</string>
   </property>
  </action>
  <action name="rotate180Act">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/rotate_180.svg</normaloff>:/images/assets/rotate_180.svg</iconset>
   </property>
   <property name="text">
    <string>Rotate 180</string>
   </property>
  </action>
  <action name="deskewAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/deskew.svg</normaloff>:/images/assets/deskew.svg</iconset>
   </property>
   <property name="text">
    <string>Deskew</string>
   </property>
  </action>
  <action name="despeckleAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/despeckle.svg</normaloff>:/images/assets/despeckle.svg</iconset>
   </property>
   <property name="text">
    <string>Despeckle</string>
   </property>
  </action>
  <action name="devoidAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/devoid.svg</normaloff>:/images/assets/devoid.svg</iconset>
   </property>
   <property name="text">
    <string>Devoid</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/zoom_in.svg</normaloff>:/images/assets/zoom_in.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/zoom_out.svg</normaloff>:/images/assets/zoom_out.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
  </action>
  <action name="fitToWindowAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/fit_window.svg</normaloff>:/images/assets/fit_window.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Fit</string>
   </property>
   <property name="toolTip">
    <string>Fit To Window</string>
   </property>
  </action>
  <action name="fillWindowAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/fill_window.svg</normaloff>:/images/assets/fill_window.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Fill</string>
   </property>
   <property name="toolTip">
    <string>Fill Window</string>
   </property>
  </action>
  <action name="fitWidthAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/fit_width.svg</normaloff>:/images/assets/fit_width.svg</iconset>
   </property>
   <property name="text">
    <string>Fit Width</string>
   </property>
  </action>
  <action name="fitHeightAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/fit_height.svg</normaloff>:/images/assets/fit_height.svg</iconset>
   </property>
   <property name="text">
    <string>Fit Height</string>
   </property>
  </action>
  <action name="pointerAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/pointer.svg</normaloff>:/images/assets/pointer.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Pointer</string>
   </property>
  </action>
  <action name="pencilAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/pencil.svg</normaloff>:/images/assets/pencil.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Pencil</string>
   </property>
  </action>
  <action name="eraserAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/eraser.svg</normaloff>:/images/assets/eraser.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Eraser</string>
   </property>
  </action>
  <action name="pix1Act">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/pix1.svg</normaloff>:/images/assets/pix1.svg</iconset>
   </property>
   <property name="text">
    <string>1px</string>
   </property>
  </action>
  <action name="pix4Act">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/pix4.svg</normaloff>:/images/assets/pix4.svg</iconset>
   </property>
   <property name="text">
    <string>4px</string>
   </property>
  </action>
  <action name="pix8Act">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/pix8.svg</normaloff>:/images/assets/pix8.svg</iconset>
   </property>
   <property name="text">
    <string>8px</string>
   </property>
  </action>
  <action name="pix12Act">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/pix12.svg</normaloff>:/images/assets/pix12.svg</iconset>
   </property>
   <property name="text">
    <string>12px</string>
   </property>
   <property name="toolTip">
    <string>12px</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="aboutQtAct">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="colorAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/fg_color.ico</normaloff>:/images/assets/fg_color.ico</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="toolTip">
    <string>Color Picker</string>
   </property>
  </action>
  <action name="undoAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/undo.svg</normaloff>:/images/assets/undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo previous image edit</string>
   </property>
  </action>
  <action name="redoAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/redo.svg</normaloff>:/images/assets/redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Undo previous undo</string>
   </property>
  </action>
  <action name="grayscaleAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/grayscale.svg</normaloff>:/images/assets/grayscale.svg</iconset>
   </property>
   <property name="text">
    <string>to Grayscale</string>
   </property>
   <property name="toolTip">
    <string>Convert to grayscale</string>
   </property>
  </action>
  <action name="binaryAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/binary.svg</normaloff>:/images/assets/binary.svg</iconset>
   </property>
   <property name="text">
    <string>to Binary</string>
   </property>
   <property name="toolTip">
    <string>Convert to binary (Otsu)</string>
   </property>
  </action>
  <action name="adaptiveBinaryAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/a_binary.svg</normaloff>:/images/assets/a_binary.svg</iconset>
   </property>
   <property name="text">
    <string>Adaptive Binary</string>
   </property>
   <property name="toolTip">
    <string>Convert to binary (Adaptive)</string>
   </property>
  </action>
  <action name="ditheredBinaryAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/d_binary.svg</normaloff>:/images/assets/d_binary.svg</iconset>
   </property>
   <property name="text">
    <string>Dithered Binary</string>
   </property>
   <property name="toolTip">
    <string>Convert to binary (Dithered)</string>
   </property>
  </action>
  <action name="blankAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/blank.svg</normaloff>:/images/assets/blank.svg</iconset>
   </property>
   <property name="text">
    <string>Blank Page</string>
   </property>
   <property name="toolTip">
    <string>Erase page and insert 'Blank'</string>
   </property>
  </action>
  <action name="fontAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/font_select.svg</normaloff>:/images/assets/font_select.svg</iconset>
   </property>
   <property name="text">
    <string>Select Font</string>
   </property>
   <property name="toolTip">
    <string>Font selection dialog</string>
   </property>
  </action>
  <action name="dropperAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/dropper.svg</normaloff>:/images/assets/dropper.svg</iconset>
   </property>
   <property name="text">
    <string>Dropper</string>
   </property>
   <property name="toolTip">
    <string>Erase all pixels with similiar color to selection</string>
   </property>
  </action>
  <action name="removeAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/remove.svg</normaloff>:/images/assets/remove.svg</iconset>
   </property>
   <property name="text">
    <string>BG Remove</string>
   </property>
   <property name="toolTip">
    <string>Erase all pixels close to white</string>
   </property>
  </action>
  <action name="floodAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/flood.svg</normaloff>:/images/assets/flood.svg</iconset>
   </property>
   <property name="text">
    <string>Flood Fill</string>
   </property>
   <property name="toolTip">
    <string>Erase all pixels with similiar color contiguous with selection</string>
   </property>
  </action>
  <action name="mirrorHorizAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/mirror_horiz.svg</normaloff>:/images/assets/mirror_horiz.svg</iconset>
   </property>
   <property name="text">
    <string>Horizontal Mirror</string>
   </property>
  </action>
  <action name="mirrorVertAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/mirror_vert.svg</normaloff>:/images/assets/mirror_vert.svg</iconset>
   </property>
   <property name="text">
    <string>Vertical Mirror</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Bookmarks</class>
   <extends>QListWidget</extends>
   <header>Bookmarks.h</header>
  </customwidget>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>Viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
